src/Raven.Server/Documents/Indexes/Static/DynamicArray.cs

```diff
@@ -13,6 +13,8 @@ public class DynamicArray : DynamicObject, IOrderedEnumerable<object>
     {
         private readonly IEnumerable<object> _inner;
 
+        private Lazy<JsonOperationContext> _context = new(() => CurrentIndexingScope.Current?.IndexContext);
```

Comment:
Why is this lazy?

Reply:
No readon, for me the option without Lazy is more dirty, pushed